tests/clone_panic.rs

```diff
@@ -0,0 +1,65 @@
+extern crate gio;
+extern crate glib;
+extern crate gtk;
```

Comment:
These circular dependencies here are not ideal, even if it's just for the tests :) Can you remove them, or move this to the gtk repo?